main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(action) {
```

Comment:
Ok makes sense. @kavanagh has apparently used it so don't want to break his stuff :p

PS: I thought demos didn't care about o-hoverable and had it `o-hoverable-on` at all times?